scripts/survey/README.md

```diff
@@ -0,0 +1,26 @@
+## Migrations
+
+#### Steps to run the script files
+
+This script is intended as a remedy to push completed surveySubmission records to kafka and it is related to the 6.0.0 branch
+
+### Step 1:
+
+    Navigate to Project Directory 
```

Comment:
@borkarsaish65 should we add from project directory to which directory to navigate?